Data-Structures/Array/TwoSum.js

```diff
@@ -0,0 +1,24 @@
+// https://leetcode.com/problems/two-sum/
+
+// Given an array of integers, return indices of the two numbers such that they add up to
+// a specific target.
+
+// You may assume that each input would have exactly one solution, and you may not use the
+// same element twice.
+
+// Example:
+// Given nums = [2, 7, 11, 15], target = 9,
+
+// Because nums[0] + nums[1] = 2 + 7 = 9,
+// return [0, 1].
+
+const TwoSum = (nums, target) => {
```

Comment:
That's a pretty naive implementation with a runtime of O(n²). Please document this in the JSDoc comment.

Reply:
Please see [the docs](https://jsdoc.app/about-getting-started.html) and use the proper `@annotations`.